src/tp206to211.jl

```diff
@@ -0,0 +1,40 @@
+# TP problem 206, 207, 208, 209, 210 and 211 in NLS format
+#
+#   Source:
+#   Klaus Schittkowski,
+#   More test examples for nonlinear programming codes,
+#   Lecture Notes in Economics and Mathematical Systems 282,
+#   Springer Verlag Berlin Heidelberg, 1987
+#   10.1007/978-3-642-61582-5
+#
+# A. Montoison, Montréal, 06/2018.
+
+export tp206, tp207, tp208, tp209, tp210, tp211
+
+"Test problem 206 in NLS format"
+function tp206(a :: Real=1, b :: Real=10, c :: Int=2; version :: String="tp206")
```

Comment:
Why `Real`? Does JuMP handle other data types than `Float64`?

Reply:
I use ```Real``` to be able to write ```a``` and  ```b``` in  ```Interger``` format too.